pkg/commands/apply/apply_run.go

```diff
@@ -0,0 +1,500 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package apply provides the Terraform apply functionality for Guardian.
+package apply
+
+import (
+	"context"
+	"errors"
+	"flag"
+	"fmt"
+	"io"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/abcxyz/guardian/pkg/flags"
+	"github.com/abcxyz/guardian/pkg/github"
+	"github.com/abcxyz/guardian/pkg/storage"
+	"github.com/abcxyz/guardian/pkg/terraform"
+	"github.com/abcxyz/guardian/pkg/util"
+	"github.com/abcxyz/pkg/cli"
+	"github.com/abcxyz/pkg/logging"
+	"github.com/sethvargo/go-githubactions"
+)
+
+const (
+	ownerReadWritePerms = 0o600
+	CommentPrefix       = "**`🔱 Guardian 🔱 APPLY`** -"
+)
+
+var _ cli.Command = (*ApplyRunCommand)(nil)
+
+// RunResult is the result of a apply operation.
+type RunResult struct {
+	commentDetails string
+}
+
+// ApplyRunCommand is a subcommand of apply and implements the cli.Command interface.
+// It performs terraform apply on the given working directory.
+type ApplyRunCommand struct {
+	cli.BaseCommand
+
+	cfg *Config
+
+	directory                 string
+	childPath                 string
+	planFilename              string
+	gitHubLogURL              string
+	computedPullRequestNumber int
+
+	flags.GitHubFlags
+	flags.RetryFlags
+
+	flagBucketName           string
+	flagCommitSHA            string
+	flagPullRequestNumber    int
+	flagAllowLockfileChanges bool
+	flagLockTimeout          time.Duration
+
+	actions         *githubactions.Action
+	githubClient    github.GitHub
```

Comment:
github -> gitHub at the beginning of the line for consistency with Git**H**ub at the end of the line.